Lib/test/test_tools/test_makefile.py

```diff
@@ -67,6 +67,9 @@ def test_makefile_test_folders(self):
                 )
                 used.append(relpath)
 
+        if sysconfig.get_config_var('WHEEL_PKG_DIR'):
```

Comment:
Perhaps this deserves a comment?

```suggestion
        # Don't check the wheel dir when Python is built --with-wheel-pkg-dir
        if sysconfig.get_config_var('WHEEL_PKG_DIR'):
```